astroquery/irsa/core.py

```diff
@@ -451,10 +451,10 @@ def _parse_coordinates(coordinates):
             warnings.warn("Coordinate string is being interpreted as an "
                           "ICRS coordinate.")
         except u.UnitsError as ex:
-            warnings.warn("Only ICRS coordinates can be entered as strings\n"
+            warnings.warn("Only ICRS coordinates can be entered as strings \n"
```

Comment:
there's no need for a space before a newline.  A period would be appropriate though.